version/version.go

```diff
@@ -2,15 +2,13 @@ package version
 
 import (
 	"fmt"
-
 	"io"
 
-	"github.com/pkg/errors"
```

Comment:
pls undo this change, bad formating:
must be 
STD lib
3rd party
custom pkgs

Reply:
Oh, weird.  My editor auto organized those.  It looks like it's alphabetical from github.  I'll try to revert.

Reply:
I see what happened.  I removed the empty space between `"github.com/pkg/errors" ` and the astro imports and it grouped them altogether.